lint/comments_checker.go

```diff
@@ -0,0 +1,33 @@
+package lint
+
+import (
+	"go/ast"
+	"regexp"
+)
+
+type commentsChecker struct {
+	ctx    *context
+	regexp *regexp.Regexp
```

Comment:
maybe s/regexp/re It conflicts with "regexp" package name.

Reply:
It can also describe actual regexp and what it supposed to match.
For example: `badCommentRE` or `commentRE`.
But that's up to you. Naming things is hard.

By the way, conflicts at field names are not a problem as you access them through object and there is no ambiguity for the compiler. So `len` is OK field name. :)

Reply:
@Quasilyte, thanks for correction. Anyway we need to rename the variable :)